resources/proxytrack.rb

```diff
@@ -22,17 +22,26 @@
     Restart=always
     RestartSec=1
     User=#{new_resource.user}
-    ExecStart=/bin/proxytrack #{new_resource.proxy_address}:#{new_resource.proxy_port} #{new_resource.icp_address}:#{new_resource.icp_port} #{new_resource.httrack_file_paths.join(' ')}
+    ExecStart=/bin/proxytrack #{new_resource.proxy_address}:#{new_resource.proxy_port} #{new_resource.icp_address}:#{new_resource.icp_port} #{new_resource.httrack_file_paths.sort.join(' ')}
 
     [Install]
     WantedBy=multi-user.target
     EOF
-    action [:create, :start, :enable]
+    action [:create]
+    notifies :restart, "service[proxytrack-#{new_resource.name}]", :immediately
```

Comment:
I don't think we need to restart this immediately.
```suggestion
    notifies :restart, "service[proxytrack-#{new_resource.name}]"
```